wger/settings.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from wger.settings_global import *
+
+# Use 'DEBUG = True' to get more details for server errors
+DEBUG = True
+TEMPLATES[0]['OPTIONS']['debug'] = True
+
+ADMINS = (
+    ('Your name', 'your_email@example.com'),
+)
+MANAGERS = ADMINS
+
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': os.getenv('NAME'),
+        'USER': os.getenv('USER'),
+        'PASSWORD': os.getenv('PASS'),
+        'HOST': os.getenv('HOST'),
+        'PORT': ''
+    }
+}
+
+# Make this unique, and don't share it with anybody.
+SECRET_KEY = '@d-bo*ihpg_mo#64ya+=d050df!gq5bum59qbf4+yh+cn+h3^c'
```

Comment:
It is discouraged to have sensitive information such as this hard coded to the system. I would encourage you to change it to an env variable

Reply:
Resolved